includes/html/print-interface.inc.php

```diff
@@ -72,6 +73,10 @@
 
 echo '</span>';
 
+$port_group_name_list = PortGroup::leftJoin('port_group_port', 'port_groups.id', '=', 'port_group_port.port_group_id')->where('port_group_port.port_id', '=', $port['port_id'])->pluck('name')->toArray() ?: ['Default'];
```

Comment:
This should use the relationship instead. `$port->group()->pluck('name');`

Reply:
```
use App/Models/Port;

$_port = Port::where('port_id', '=', $port['port_id']);
$port_group_name_list = $_port->groups()->pluck('name');

```

results in :
"
Call to undefined method Illuminate\Database\Eloquent\Builder::groups()
"

ideas?

Reply:
You did not fetch from the database.  You need a ->get() after the where.

Better:
```php
Port::find($port['port_id'])->groups->pluck('name');
```

Would be better if you can prefetch all the port models with groups instead of causing two sql queries per port.

Reply:
aahh , details :-)  thanks